sql/src/main/java/com/rilldata/calcite/extensions/SqlCreateMetric.java

```diff
@@ -108,7 +108,7 @@ private static Map<String, SqlNode> createMapFromList(SqlNodeList sqlNodes) thro
     for (SqlNode sqlNode : sqlNodes) {
       String name = sqlNode.accept(simpleIdentifierExtractor);
       if (name == null) {
-        throw new ParseException(String.format("Cannot find identifier for node %s", sqlNode));
+        throw new ParseException(String.format("Please provide an alias for %s", sqlNode));
       }
```

Comment:
Would it make sense to provide default aliases (like `measure1`, `measure2`, etc.) instead? (Note that I added this quickly, might be misinterpreting the context)

Reply:
This is for using aliases for measures in the metrics view model. I think users should specify them explicitly as they will use these aliases in the sql queries in which they will use the defined model.

Reply:
I was thinking of the case where a user just writes the metrics view, and then a dashboard is generated from the metrics view. But your argument also makes sense. Let's stick with your way now and see if any users complain.